src/devices/video/nereid.cpp

```diff
@@ -0,0 +1,101 @@
+// license:BSD-3-Clause
+// copyright-holders:Sven Schnelle
+
+#include "emu.h"
+#include "nereid.h"
+
+//#define VERBOSE 1
+#include "logmacro.h"
+
+DEFINE_DEVICE_TYPE(NEREID, nereid_device, "nereid", "HP Nereid ASIC")
+
+nereid_device::nereid_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock)
+	: device_t(mconfig, type, tag, owner, clock),
```

Comment:
Please use either colon and comma at end of line, or colon and comma at beginning of line for initialiser lists - not a mixture with colon at beginning but comma at end.